wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -0,0 +1,477 @@
+                            Icon(
+                                imageVector = Icons.Filled.ArrowBack,
+                                contentDescription = "Back Arrow"
+                            )
+                        }
+                    },
+                    scrollBehavior = scrollBehavior,
+                )
+            },
+        ) { innerPadding ->
+            Column(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .padding(innerPadding),
+                verticalArrangement = Arrangement.Center,
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.Center
+                ) {
+                    Text(
+                        modifier = Modifier.padding(8.dp),
+                        text = "TODO: About Screen"
+                    )
+                }
+            }
+        }
+    }
+
+    @Composable
+    fun AddToWalletScreen(navigation: NavHostController) {
+        val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior(rememberTopAppBarState())
+        Scaffold(
+            topBar = {
+                CenterAlignedTopAppBar(
+                    colors = TopAppBarDefaults.topAppBarColors(
+                        containerColor = MaterialTheme.colorScheme.primaryContainer,
+                        titleContentColor = MaterialTheme.colorScheme.primary,
+                    ),
+                    title = {
+                        Text(
+                            "Add to Wallet",
+                            maxLines = 1,
+                            overflow = TextOverflow.Ellipsis
+                        )
+                    },
+                    navigationIcon = {
+                        IconButton(
+                            onClick = {
+                                navigation.popBackStack()
+                            }
+                        ) {
+                            Icon(
+                                imageVector = Icons.Filled.ArrowBack,
+                                contentDescription = "Back Arrow"
+                            )
+                        }
+                    },
+                    scrollBehavior = scrollBehavior,
+                )
+            },
+        ) { innerPadding ->
+            Column(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .padding(innerPadding),
+                verticalArrangement = Arrangement.Center,
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.Center
+                ) {
+                    Button(onClick = {
+                        if (application.addSelfsignedMdl()) {
+                            navigation.popBackStack()
+                        } else {
+                            Toast.makeText(applicationContext,
```

Comment:
alternatives: 
- snackbar that dismisses itself after 5? or 10? seconds to user has a chance to see this message
- popup modal/dialog with acknowledgement from user in the event they don't see the toast or snackbar on the bottom, tapping on "OK" dismisses it

Reply:
I wouldn't worry about this. This code here is very temporary and will go away once we stand up a proper `IssuingAuthority` interface with a dedicated `IssuingAuthoritySelfSignedMdl` implementation (as well as other implementations).